src/entities/protocols/looksRareV2.ts

```diff
@@ -101,29 +112,37 @@ export class LooksRareV2Trade extends NFTTrade<LooksRareV2Data> {
     return total
   }
 
-  private refactorAPIData(data: LooksRareV2Data): {
-    takerBid: TakerOrder
-    makerOrder: MakerOrder
-    makerSignature: string
-    value: BigNumber
-    merkleTree: MerkleTree
+  private refactorAPIData(orders: LooksRareV2Data[]): {
+    takerBids: TakerOrder[]
+    makerOrders: MakerOrder[]
+    makerSignatures: string[]
+    totalValue: BigNumber
+    merkleTrees: MerkleTree[]
   } {
-    const makerOrder: MakerOrder = { ...data.apiOrder }
+    let takerBids: TakerOrder[] = []
+    let makerOrders: MakerOrder[] = []
+    let makerSignatures: string[] = []
+    let totalValue: BigNumber = BigNumber.from(0)
+    let merkleTrees: MerkleTree[] = []
 
-    const makerSignature: string = data.apiOrder.signature
+    orders.forEach((order) => {
+      makerOrders.push({ ...order.apiOrder })
 
-    const takerBid: TakerOrder = {
-      recipient: data.taker,
-      additionalParameters: '0x',
-    }
+      makerSignatures.push(order.apiOrder.signature)
 
-    const value: BigNumber = BigNumber.from(data.apiOrder.price)
+      takerBids.push({
+        recipient: order.taker,
+        additionalParameters: '0x',
```

Comment:
Is it possible that there are additionalParameters specified in the `order` object returned from the API?

Reply:
the additional parameters are never actually used for anything onchain... have messaged LooksRare team to ask what theyre for